gui/wxpython/gui_core/toolbars.py

```diff
@@ -30,6 +30,11 @@
 
 from grass.pydispatch.signal import Signal
 
+try:
+    from agw import aui
+except ImportError:
+    import wx.lib.agw.aui as aui
+
```

Comment:
Please check current import style and update if needed

Reply:
I checked in wxPython Demo version 4.1.1. ale it's used like that